File: 백준/Bronze/25304. 영수증/영수증.cc
#include <iostream>

using namespace std;

int main(void) {
    int total_price, total_num, price, num, sum = 0;
    
    cin >> total_price >> total_num;
    
    for (int i = 0; i < total_num; i++) {
        cin >> price >> num;
        sum += price * num;
    }
    
    if (total_price == sum) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
    return 0;
}

File: 백준/Bronze/25304. 영수증/영수증.py
total_price = int(input())
goods_num = int(input())

result_price = 0
for _ in range(goods_num) :
    goods_price, num = map(int, input().split())
    result_price += goods_price * num
    
if (total_price == result_price) :
    print("Yes")
else :
    print("No")